Snail/Variable.swift

```diff
@@ -31,6 +31,12 @@ public class Variable<T> {
         return subject
     }
 
+    public func bind(to variable: Variable<T>) {
+        variable.asObservable().subscribe(onNext: { [weak self] value in
```

Comment:
this only covers the happy path, lets also add error and done

Reply:
@russellbstephens the extension is on `Variable` and takes another `Variable` as parameter, not an `Observable` = there is no way to send `.error` nor `.done` 👍

Reply:
👍